Sources/MapboxCoreNavigation/Router.swift

```diff
@@ -13,9 +13,17 @@ public protocol RouterDataSource: AnyObject {
 }
 
 /**
- A route and its index in a `RouteResponse` that sorts routes from most optimal to least optimal.
+ A `RouteResponse` object that sorts routes from most optimal to least optimal and selected route index in it.
  */
-public typealias IndexedRoute = (Route, Int)
+public struct IndexedRouteResponse {
+    public let routeResponse: RouteResponse
+    public let routeIndex: Int
```

Comment:
Remember to add documentation comments about these properties and also the initializer below so that jazzy recognizes them.